ci/jenkins/jobs/macros.yaml

```diff
@@ -167,3 +167,15 @@
             echo "Whole Conformance Test failed!"
           fi
           exit $((TEST_FAIL_E2E + TEST_FAIL_NP + TEST_FAIL_CONFORMANCE))
+
+- builder:
+    name: builder-e2e-vm
+    builders:
+        - shell: |-
+          #!/bin/bash
+          set -e
+          TEST_FAIL_E2E=0
+          TEST_FAIL_NP=0
+          TEST_FAIL_CONFORMANCE=0
```

Comment:
TEST_FAIL_E2E=0
TEST_FAIL_NP=0
TEST_FAIL_CONFORMANCE=0
Remove if you don't need them in your script

Reply:
Sure